doc/specifications/content_edit.md

```diff
@@ -0,0 +1,58 @@
+# Symfony forms content edit
+
+## Synopsis
+repository-forms supports creating content with and without an intermediate draft. This describes
+editing with drafts.
+
+Editing content implies creating a new version of an existing content, rendering a form with the
+previous version's fields values, and allowing the user to save the draft, cancel or publish the
+new version.
+
+
+### Creating a draft from an existing version
+
+> A route that displays a form allowing to create a draft from an existing content item.
```

Comment:
You mean "...allowing *you* to..."? It's common and works fine either way I think. I wouldn't change it unless a native speaker says it's borked :)